assets/js/paddle-affiliate-wp.js

```diff
@@ -1,46 +1,46 @@
 function appseroSetupPaddle() {
     Paddle.Setup({
-        vendor: appseroPaddleAffWP.vendor_id,
-        eventCallback: function(data) {
+        vendor: parseInt(appseroPaddleAffWP.vendor_id),
+        eventCallback: function (data) {
             appseroPaddleEvent(data);
         },
     });
 }
 
-jQuery( document ).ready( function() {
+jQuery(document).ready(function () {
     appseroSetupAffiliatePaddle();
 });
 
 function appseroSetupAffiliatePaddle() {
-    if( window.Paddle === undefined || !window.Paddle ) {
+    if (window.Paddle === undefined || !window.Paddle) {
         return;
     }
 
-    if( window.PaddleCompletedSetup === undefined || !window.PaddleCompletedSetup ) {
+    if (
+        window.PaddleCompletedSetup === undefined ||
+        !window.PaddleCompletedSetup
+    ) {
         appseroSetupPaddle();
         return;
     }
 
     Paddle.Options({
-        eventCallback: function(data) {
+        eventCallback: function (data) {
             appseroPaddleEvent(data);
-        }
+        },
     });
 }
 
 function appseroPaddleEvent(data) {
-
-    if( data.event === 'Checkout.Complete' ) {
+    if (data.event === "Checkout.Complete") {
```

Comment:
May we know the reason for replacing single quote with double quote?

Reply:
It was just a eslint fixing